server/src/immich/api-v1/asset/asset.service.ts

```diff
@@ -207,12 +207,13 @@ export class AssetService {
 
     const allowExif = this.getExifPermission(authUser);
     const asset = await this._assetRepository.getById(assetId);
+    let data = allowExif ? mapAsset(asset) : mapAssetWithoutExif(asset);
 
-    if (allowExif) {
-      return mapAsset(asset);
-    } else {
-      return mapAssetWithoutExif(asset);
+    if (data.ownerId !== authUser.id) {
+      delete data.people;
```

Comment:
Should we set it to an empty list instead or delete it?

Reply:
`people` is optional so removing it is valid, but it's probably better to set it to an empty list